dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImpl.java

```diff
@@ -1683,6 +1689,139 @@ protected SearchHits indexSearch(final String query, final int limit, final int
 
     }
 
+//    @Override
+    protected PaginatedArrayList<ContentletSearch> indexSearchScroll(final String query, String sortBy) {
```

Comment:
#CloseDBIfOpened?

Reply:
Original `indexSearch` (not scroll) method doesn't have it. The only db operation it does is a call to get the indices `info=APILocator.getIndiciesAPI().loadIndicies();` which already has the annotation